### ISSUE TITLE
Anything I try to launch in Sandbox generates SBIE2101 Error

### ISSUE THREAD

**Issue by ThankYou5** [sandboxie-plus/Sandboxie#67]
Windows 10 Pro x64
SB 5.41.1

One of my main needs for SB is to be able to right click on executable files and to run them in a sandbox... especially to install programs in the sandbox I want to try out..

But many versions ago this stopped working.  Now everything I try to run this way generates an error similar to below.  Still doing it with v5.41.1

I reported this in the Sophos forum and several people responded they have the same problem.  I can't use SB for this anymore.

**SBIE2101 Object name not found: \Sessions\1\BaseNamedObjects\ISWWH_BEACON@197c@EFR-controller, error CreateEvent (C0000022) access=001F0003 initialized=1
SBIE2314 Canceling process SandboxieBITS.exe [6524 / 7]**

**Comment by DavidXanatos** [comment]
hmm... for me that still works... I cant fix something I cant reproduce.
what other security software are you using?

**Comment by ThankYou5** [comment]
Hi, sorry I can't offer any more information for now.  As a matter of fact v5.41.1 (the first opensource version I've tried) didn't work out well for me.  In addition to the problem I described here, which I already had with previous SB versions, I also had other new problems with this build.

I have always run Chrome in a sandbox, and it was working fine before I installed v5.41.1, but after I posted this thread, suddenly that was generating SB errors when I launched it too.

SBIE2101 Object name not found: \Sessions\1\BaseNamedObjects\ISWWH_BEACON@1af0@EFR-controller, error CreateEvent (C0000022) access=001F0003 initialized=1
SBIE2314 Canceling process SandboxieRpcSs.exe [6896 / 7]
SBIE2204 Cannot start sandboxed service RpcSs (1)
SBIE2204 Cannot start sandboxed service DcomLaunch (-4)

So I installed SB v5.31.6 over it which I have heard is a good stable version and I am able to use Chrome in the sandbox again with no errors.  It's not clear whether or not it fixed the issue I posted about in the OP though.

The only security software (other than Zonealarm firewall) I am using is stock Windows Defender.

**Comment by ThankYou5** [comment]
Follow-up.

In the thread I started in the Sophos forum on this topic someone also mentioned that dropping back to v5.31.6 fixed it for them.  They did not receive the error listed here in the OP using that version.

I just tested it and found that it also fixes that problem for me.  I can now right click on .exe installers in explorer and install them in the sandbox without generating the error.  I tested it with a few of the same installers that did not work before.  It appears to be working again.

I do not know if v5.31.6 is the last version that works or if the issue didn't start until some other version after that.  But my test indicates that the issue I posted in the OP didn't start until some version after this one.

Thanks again for your response.  If I can add anything helpful, I will come back and post it,

**Comment by DavidXanatos** [comment]
can you provide some of these installers

**Comment by ThankYou5** [comment]
Hi DX

Since I wrote this, I sadly gave up on SB.  I used it for many years and never had any problems.  The problems started when I started using Windows 10.  There are so many bugs now, many of them left over from the Sophos days, that I reluctantly gave up on it.  I just got discouraged after bug on top of bug that they will never be fixed.  It broke my heart because SB was one of my all time favorite programs.

I used it to run my browser in a sandbox and to test new software in without screwing up my file system and registry.  But starting when Sophos was still hosting SBIE, the bugs began to pile up, new software testing was completely broken and I was doing so many workarounds just to use Chrome in the sandbox that it wasn't worth it.  And I wasn't even sure the sandbox was working because I had to ignore so many errors.  Finally a couple of months ago, I decided to give up.  My thinking now is to install a virtual machine instead (and hope I can get that to work!).

I still wish that I could get SBIE to work, but the problems and errors got so bad I can't even keep up with even tracking them.  And some of the bugs are so old I don't think they are ever going to get fixed.  So I gave up.  I am running Chrome without the sandbox and it works fine, but I hate running it this way.  Right now I have no choice.  It's way too klugy in SBIE, if I can get it to run at all.

Anyway, I still wish that I could use SBIE, but it feels like I am out of options.

Here are the two installers I mentioned in this thread ... I just found them on the Internet and downloaded them. \

[Sandboxie 5.31.6 & 5.41.1 Installers.zip](https://github.com/sandboxie-plus/Sandboxie/files/5765624/Sandboxie.5.31.6.5.41.1.Installers.zip)

**Comment by DavidXanatos** [comment]
@ThankYou5 those are quite old you should try a mew recent build:
https://github.com/sandboxie-plus/Sandboxie/releases/download/v0.5.3b/SandboxieInstall64-v5.45.2.exe
Between what you tested and this oen a few major window w0 2002+ compatibility bugs were fixed
you may also want to try the plus build with a greatly improved UI: https://github.com/sandboxie-plus/Sandboxie/releases/download/v0.5.3b/Sandboxie-Plus-x64-v0.5.3.exe

**Comment by ThankYou5** [comment]
Thanks for your reply DX.

Yes, I am aware that the versions I mentioned here are old.  I dropped back to those older versions because the newer versions didn't work.  5.31.6 worked better after dropping back, but even that had issues with running Chrome in the box for which I needed to do workarounds and I couldn't install programs in the Sandbox.

I tried a recent version of SBIE+ and although I really like what you did to the interface, I had the same old errors that I've had since the Sophos days (plus a few more). 

Is SBIE 5.45.2 the last Sophos release?  Has anyone else come up with newer SBIE versions/forks aside from SBIE+ since Sophos released it?  I know there was talk about it, but the last time I looked it was still just talk.

If SBIE 5.45.2 is the final Sophos version, I already tried that and it was loaded with errors and problems for me.

Thanks again for your help and your great work!
